multifidelity_end2end.py

```diff
@@ -147,13 +207,21 @@ def main():
         devices=1,
         max_epochs=args.num_epochs,
     )
-    trainer.fit(mpnn, train_loader, val_loader)
 
-    preds = trainer.predict(mpnn, test_loader)
+    if args.model_type == "transfer":
+        
+        trainer.fit(mpnn, lf_train_loader, lf_val_loader)
```

Comment:
I think we should also do `trainer.predict` after the fitting on the LF data. We should do separate predictions on the `hf_test_loader` and `lf_test_loader`.